### PR TITLE
716 frontend change proposedsolutionform to use nersuccessbutton

### PR DESCRIPTION
**Pull request by 5jayarama**
## Changes

Changed button to NER button

## Notes
used ticket 731 as reference

## Screenshots
image
image
image
image

## Checklist

It can be helpful to check the `Checks` and `Files changed` tabs.
Please review the [contributor guide](https://nerdocs.atlassian.net/wiki/spaces/NER/pages/8060929/Software+Contributor+Guide) and reach out to your Tech Lead if anything is unclear.
Please request reviewers and ping on slack only after you've gone through this whole checklist.

- [x] All commits are tagged with the ticket number
- [x] No linting errors / newline at end of file warnings
- [x] All code follows repository-configured prettier formatting
- [x] No merge conflicts
- [x] All checks passing
- [x] Screenshots of UI changes (see Screenshots section)
- [x] Remove any non-applicable sections of this template
- [x] Assign the PR to yourself
- [x] No `yarn.lock` changes (unless dependencies have changed)
- [x] Request reviewers & ping on Slack
- [x] PR is linked to the ticket (fill in the closes line below)

Closes # (issue #)
#716

### CODE REVIEW

File: src/frontend/src/pages/ChangeRequestDetailPage/ProposedSolutionForm.tsx [hunk 1]
```diff
@@ -178,16 +179,24 @@ const ProposedSolutionForm: React.FC<ProposedSolutionFormProps> = ({
...
+                sx={{ textTransform: 'none', fontSize: 16, marginTop: 3 }}
+              >
+                Cancel
+              </NERFailButton>
+              <NERSuccessButton
                 color="success"
                 variant="contained"
                 type="submit"
                 form="individual-proposed-solution-form"
                 sx={{ textTransform: 'none', fontSize: 16, marginTop: 3 }}
```
Comment (Peyton-McKee) [inline src/frontend/src/pages/ChangeRequestDetailPage/ProposedSolutionForm.tsx:196]: we want a margin inbetween the buttons so can you add spacing of two here

**Review by Peyton-McKee — CHANGES_REQUESTED** [review]
Good job on the code! but you have yarn lock changes that you need to undo, and you also have a file called HW7.lean in your repo that you need to remove. Also can you link your pr to your ticket by filling out the closes number with your ticket number.

**Comment by anthonybernardi** [comment]
pls redo this PR with correctly tagged branch name (need the #), correctly tagged commits (need "#716: blah blah"), no yarn.lock changes, linking the ticket to the pr (type "closes #716" exactly like that), and no homework files submitted 😆 a

**Comment by anthonybernardi** [comment]
also there shouldn't have been conflicts with the proposed solution file so you probably need to pull develop again

**Pull request closed without merging**